aks/le-prod-cert-issuer.yaml.template

```diff
@@ -12,7 +12,7 @@ spec:
       # The secret that holds the generated private key used to communicate with Let's Encrypt
       name: letsencrypt-prod-account-key
     dns01:
-    providers:
+      providers:
```

Comment:
Ah, that infamous indentation issue which took us half a day to figure out... :-)

Reply:
Yes :)